source/features/useful-forks.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import {LinkExternalIcon} from '@primer/octicons-react';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function init(): void {
+	const downloadUrl = new URL('https://useful-forks.github.io/?repo=' + pageDetect.utils.getRepoURL());
```

Comment:
```suggestion
	const downloadUrl = new URL('https://useful-forks.github.io');
	downloadUrl.searchParams.set('repo', pageDetect.utils.getRepoURL());
```

Reply:
Okay, I'm now using `getRepo()`.

However, I was aware that `downloadUrl.searchParams.set` existed, but it causes a bug, so I went for manual concatenation.

The thing encodes the special characters in HTML format, likeso:

![image](https://user-images.githubusercontent.com/38117856/107280451-645c3c80-6a26-11eb-8198-665c25b3b927.png)

I would expect `payne911/PieMenu`, but get `payne911%2FPieMenu` instead.

Reply:
> I would expect `payne911/PieMenu`, but get `payne911%2FPieMenu` instead.

That's because you should use the same thing in useful-forks as well. The URL parameters should be correctly encoded/unencoded. In this case:

```js
const repo = new URLSearchParams(location.search).get('repo');

Reply:
All fixed! Thanks.